module/VuFind/src/VuFind/ILS/Driver/SierraRest.php

```diff
@@ -1733,8 +1734,8 @@ protected function getItemStatusesForBib($id)
             if (!empty($holdingsResult['entries'])) {
                 foreach ($holdingsResult['entries'] as $entry) {
                     $location = '';
-                    foreach ($entry['fixedFields'] as $field) {
-                        if ('LOCATION' === $field['label']) {
+                    foreach ($entry['fixedFields'] as $code => $field) {
+                        if ($code === 40 || $field['label'] === 'LOCATION') {
```

Comment:
I assume "40" is some kind of system-level identifier; would it be worth defining as a class constant for clarity?

Reply:
Yes, I added a const.